docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+---
+title: TLS for Kubernetes
+order: 0
+---
+
+# TLS for Kubernetes
+
+Talking over the internet with no encryption is something unthinkable these days. With Redpanda you can help secure your data by enabling [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) across your clusters, with just a few configurations. 
+
+If you haven’t, please check our guide about [Encryption, authorization & authentication](https://vectorized.io/docs/acls/).
+
+Here we will dive deeper into how we can configure TLS inside a Kubernetes cluster.
+
+The Custom Resource Definition (CRD) of a Redpanda cluster includes three APIs: Kafka, Admin and Pandaproxy APIs. Each API can support TLS for server authentication and mTLS (Mutual TLS) for client authentication.
+
+TLS allows us to establish a secure connection between a client and a broker in which communication is encrypted. 
+
+Mutual TLS (mTLS), or 2-way TLS, is an authentication method in which a client attempting to establish a connection with the server (broker) must present its certificate. This way the server can decide whether the client can be trusted.
+
+## Getting started
+
+To demonstrate how to get up and running with TLS in Kubernetes, we’ll use a sample `yaml` configuration file for a one-node cluster.
+
+To enable TLS we need to add a listener entry under the API fields in our yaml configuration file, and then specify the subfield `tls` where we can enable TLS.
+
+For example, check that `kafkaApi` now have a new flag that says `tls: enabled: true`.
+
+Save this file as `yaml`:
+
+```yaml
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+ name: cluster-sample-tls
+spec:
+ image: "vectorized/redpanda"
+ version: "latest"
+ replicas: 1
+ resources:
+   requests:
+     cpu: 1
+     memory: 1.2G
+   limits:
+     cpu: 1
+     memory: 1.2G
+ configuration:
+   rpcServer:
+     port: 33145
+   kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+   pandaproxyApi:
+    - port: 8082
+      tls:
+        enabled: true
+   adminApi:
+   - port: 9644
+   developerMode: true
+```
+
+We’ll use the Kafka API to go through the TLS configuration, but you can use similar settings to set up TLS for the other APIs. For this guide, we assume external connectivity is disabled unless otherwise stated.
+
+The operator supports a single listener with TLS per API. Additionally, if you have two listeners for the same API and one of them has external connectivity enabled, the operator assumes that the latter listener has TLS enabled.
+
+Providing an issuer or certificate to the operator is only available for the Kafka API. 
+
+### Client Auth (mTLS)
+
+When you enable TLS, the client has the option of verifying the identity of the server, but to do this the client must have the CA’s certificate available in a trusted store.
```

Comment:
my comment on line 18 is referring to this.

Reply:
@coral-waters do you think we should edit this part the new suggestion as well?

Reply:
I don't know. I don't know where the first part is anymore.

Reply:
https://github.com/vectorizedio/redpanda/pull/3341#discussion_r774169125